pwdsafety/pwdsafety.tape

```diff
@@ -57,11 +57,12 @@ Output pwdsafety.gif
 Require echo
 
 Set Shell "bash"
-Set FontSize 14
+Set FontSize 13
 Set Width 1200
 Set Height 600
 
 Type "pwdsafety" Sleep 1s  Enter
 Sleep 2s
 Type "password123" Sleep 1s  Enter
 Sleep 4s
+PageUp@3
```

Comment:
literally it doesn't do anything ahahah okay... let's remove the `PageUp` command and I'll merge this